script/gen_requirements_all.py

```diff
@@ -73,6 +73,7 @@
 # want to ensure we have wheels built.
```

Comment:
This PR changed the permissions for this file to 644 from 755. This is a script and it should be executable.

Reply:
Fixed in https://github.com/home-assistant/core/pull/85745.

Reply:
That's a new one. Sorry I missed that. Thanks for fixing 👍

Reply:
Figured out how I missed it....GitHub app doesn't show permission changes

Reply:
I opened report #1964196 for the app

Reply:
In the android app the indentation is completely missing for non changed lines. That's the show stopper for me with the app.